### ISSUE TITLE
rename lagoon-build-deploy to lagoon-remote-controller

### ISSUE THREAD

**Issue by shreddedbacon** [uselagoon/lagoon-charts#472]
Since this can cause confusion with things being named similarly (build-deploy-tool / build-deploy-image), I suggest renaming the lagoon-build-deploy chart to lagoon-remote-controller as this is what the chart installs.

**Comment by desdrury** [comment]
> 

Agreed.  I was just showing a couple of our guys yesterday the release details for Lagoon Core and the question of `build-deploy-tool` / `lagoon-build-deploy` came up 😀

**Comment by tobybellwood** [comment]
yup - agreed - now to work out how to do it without causing everyone headaches!
